.env.default

```diff
@@ -27,3 +27,7 @@ LOGGING_LEVEL=warn
 LOGGING_PROFILING_ENABLED=false
 
 COMPOSE_PROJECT_NAME=cherrytwist-demo
+
+SERVICE_ENDPOINT_PORT=3002
```

Comment:
Should these env vars not have names to show they are for the Simple Auth service?

Reply:
Yes and no - they are configured in the context of their own service and shouldn't be aware of the larger ecosystem they will be part of. This should be addressed on a system level and merits a larger discussion (otherwise our env variables will be each 50+ characters...).